src/core/Akka.Cluster/VectorClock.cs

```diff
@@ -172,6 +172,32 @@ public bool IsSameAs(VectorClock that)
             return CompareOnlyTo(that, Ordering.Same) == Ordering.Same;
         }
 
+        public static bool operator >(VectorClock left, VectorClock right)
```

Comment:
I like the operators a lot better than the methods we were using before. Helps readability.